docs/faq/state-tree.md

```diff
@@ -14,8 +14,8 @@ resource's reducer.
 
 Yes, you can. The only requirement is that you don't change the structure of the
 state that you start out with: make sure that `resources` remains an Array,
```

Comment:
is `resources` an Array? I thought it was an Object now

Reply:
Good call. I’ll update this.